receiver/dockerstatsreceiver/factory.go

```diff
@@ -35,9 +36,22 @@ func createDefaultConfig() component.Config {
 			Timeout:          scs.Timeout,
 		},
 		MetricsBuilderConfig: metadata.DefaultMetricsBuilderConfig(),
+		MinDockerRetryWait:   time.Second,
+		MaxDockerRetryWait:   10 * time.Second,
+		Logs:                 EventsConfig{},
```

Comment:
Will this collect events automatically? I think it should be off-by-default as to not affect existing deployments with the receiver running

Reply:
Or will the log not be active unless they put it into a log pipeline... Good to know either way

Reply:
the latter, i'd assume. because pipelines are created in the config like this:
```
pipelines:
    logs:
      receivers: ...
      exporters: ...
      processors: ...
 ```
 
 so there'd be no reason to create a logs receiver unless it was used in a pipeline, even if the config called for one.